doc/source/user_guide/window.rst

```diff
@@ -299,6 +299,38 @@ forward-looking rolling window, and we can use it as follows:
    indexer = FixedForwardWindowIndexer(window_size=2)
    df.rolling(indexer, min_periods=1).sum()
 
+Reverse rolling window
```

Comment:
If this example has the same result as the `FixedForwardWindowIndexer` you added in this section, you can just continue in the section above and show that it's equivalent to the code block you added here.

Reply:
Okay, sure.